fortex/health/readers/CT_image_reader.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2022 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This class is designed to read Xray images from given image path.
+
+    For more information for the dataset, visit:
+      https://data.mendeley.com/datasets/jctsfj2sfn/1
+    """
+import os
+from typing import Any, Iterator
+from forte.data.data_pack import DataPack
+from forte.data.base_reader import PackReader
+from forte.data.data_utils_io import dataset_path_iterator
+import pydicom
+
+
+class CTimageReader(PackReader):
+    r""":class:`CTimageReader` is designed to read CT image files from a given folder."""
```

Comment:
We should describe this more clearly, what types of data, what's the folder structure?